app/controllers/api/v1/transactions_controller.rb

```diff
@@ -0,0 +1,45 @@
+module Api::V1
+  class TransactionsController < OrganizationController
+    before_action :set_transaction,  only: [:show, :edit, :update, :destroy]
+
+    def index
+      @transactions = current_organization.transactions.page(params[:page]).per(30)
+    end
+
+    def show
+    end
+
+    def create
+      @transaction = current_organization.bank_accounts.find_by(id: transaction_params[:bank_account_id]).transactions.build transaction_params
```

Comment:
@yunixon I think checking for organization will be better to move to transaction_policy.